helper-cli/src/main/kotlin/commands/packagecuration/CreateCommand.kt

```diff
@@ -60,12 +60,19 @@ internal class CreateCommand : CliktCommand(
             throw UsageError("File ${outputFile.absolutePath} already exists.")
         }
 
+        val mapper = createBlockYamlMapper()
+
         try {
+            val text = mapper.writerWithDefaultPrettyPrinter().writeValueAsString(
+                listOf(
+                    // A block comment without any text is not valid in yaml. Therefore add a dummy command to force a
+                    // block comment.
+                    PackageCuration(packageId, PackageCurationData(comment = "Curation comment.\n"))
```

Comment:
Why the `\n` at the end?

Reply:
Ah, for the same reason as [below](https://github.com/oss-review-toolkit/ort/pull/4179/files#diff-b78e8fca71542309a4a90ff9c1f29927ec8d341894ff60f102efc9eb96bf0f58R75-R76)? Then we should probably also copy the comment.

Reply:
Yes, it's the same reason.
I did add the reason for the `\n` to the comment.